### PR TITLE
Fix task failure when multiple attempts are run

### PR DESCRIPTION
**Pull request by aballiet**
Fixes https://github.com/kestra-io/plugin-airbyte/issues/33

Currently if Airbyte sync fails and ends up with status `INCOMPLETE`, flow will return failed even though Airbyte is triggering a new attempt which could succeed.

Even worse, duration calculation can end up with negative value and make the plugin fails.

State machine of Airbyte Job Status is available [here](https://docs.airbyte.com/understanding-airbyte/jobs/#sync-job-state-machine).

state-machine

In order to fix, 
- removed `INCOMPLETE` status from the list of Ending ones.
- removed duration computation to make sure no error will be thrown. Rather display the number of attempts.
- Warn user when sync is running a new attempt

Also : 
- Allowed extra properties in payload like `enabledStreams` which is returned in Airbyte `0.50.13`

### CODE REVIEW

File: src/main/java/io/kestra/plugin/airbyte/models/Job.java [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 import lombok.Value;
 import lombok.experimental.SuperBuilder;
 import lombok.extern.jackson.Jacksonized;
-
 import java.time.Instant;
 
 @Value
 @Jacksonized
 @SuperBuilder
+@JsonIgnoreProperties(ignoreUnknown = true)
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/airbyte/models/Job.java:12]: Why is it needed?

**Review by loicmathieu — CHANGES_REQUESTED** [review]

File: src/main/java/io/kestra/plugin/airbyte/models/Job.java (hunk 1 above)
Comment (loicmathieu) [reply src/main/java/io/kestra/plugin/airbyte/models/Job.java:12]: OK, strange that it's not the default.

**Review by loicmathieu — APPROVED** [review]

File: src/main/java/io/kestra/plugin/airbyte/models/Job.java (hunk 1 above)
Comment (aballiet) [reply src/main/java/io/kestra/plugin/airbyte/models/Job.java:12]: Ignoring extra fields ?

**Comment by loicmathieu** [comment]
Thanks a lot!

**Pull request merged**